### PR TITLE
docs(button): move to the new docs site

### PR DESCRIPTION
**Pull request by gergelyke**

### CODE REVIEW

File: documentation-site/static/examples/button/with-enchancers.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {Button} from 'baseui/button';
+
+function CloudComponent() {
```
Comment (chasestarr) [inline documentation-site/static/examples/button/with-enchancers.js]: can we replace this cloud component with an icon component? this code feels like noise that does not contribute to explaining the concept

File: documentation-site/routes.js [hunk 2]
```diff
@@ -43,12 +43,16 @@ const routes = [
     text: 'Components',
     children: [
       {
-        text: 'Data Entry',
+        text: 'Basic Inputs',
```
Comment (nadiia) [inline documentation-site/routes.js]: It says `Basic Inputs` but contains a button example below. We should split button example into a separate child of the 'Components' instead.

Comment (gergelyke) [reply documentation-site/routes.js]: the idea came for Fabric UI, and how they categorized components: https://developer.microsoft.com/en-us/fabric#/components/button

what do you think, would it make sense to follow that?

Comment (nadiia) [reply documentation-site/routes.js]: Oh I see, `Inputs` might be confusing, or at least at first. Other suggestion here can be `Basic controls`. No objections to using the current name tho.

**Comment by gergelyke** [comment]
currently blocked by https://github.com/atlassian/extract-react-types/issues/41

**Comment by tajo** [comment]
Looks great! I noticed there is FOUC when the page is being loaded. And it happens in both dev and prod modes. Is that fixable?

**Comment by gergelyke** [comment]
@tajo oh yeah, I see - I'll look into that in a separate PR

**Pull request merged**